includes/modules/payment/paypaldp.php

```diff
@@ -237,8 +237,8 @@ function update_status() {
 
     // Purchase amount
     if ($this->enabled) {
-      // module cannot be used for purchase > $10,000 USD
-      $order_amount = $this->calc_order_amount($order->info['total'], 'USD');
+      // module cannot be used for purchase > $10,000 USD equiv
+      $order_amount = $this->calc_order_amount($order->info['total'], 'USD', false);
```

Comment:
This `false` was added to ensure commas aren't inserted into this number, so that the comparison on the next line works correctly.